examples/gno.land/p/demo/airdrop/merkle-airdrop.gno

```diff
@@ -0,0 +1,88 @@
+package airdrop
+
+import (
+	"crypto/sha256"
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"std"
+
+	"gno.land/p/demo/avl"
+	"gno.land/p/demo/grc/grc20"
+	"gno.land/p/demo/merkle"
+)
+
+var (
+	ErrAlreadyClaimed = errors.New("already claimed")
+	ErrInvalidProof   = errors.New("invalid merkle proof")
+)
+
+type AirdropData struct {
+	Address std.Address
+	// TODO: use std.Coin
```

Comment:
not compatible with grc20; if your airdrop package is dedicated for grc20, you can keep `amount uint`